packages/client/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
ERROR in vendor.2ee74da476daa4484836.bundle.js from UglifyJs
Unexpected character '`' [/home/ubuntu/CesiumFPS/packages/client/node_modules/angular-cesium/bundles/angular-cesium.umd.js:5461,0][vendor.2ee74da476daa4484836.bundle.js:136923,30]